### ISSUE TITLE
Copy URL to clipboard on server start

### ISSUE THREAD

**Issue by chriscalo** [chriscalo/express-start#4]
It would be nice to be able to just paste the URL into a browser window once the server has started.

StackOverflow question:
https://stackoverflow.com/questions/7778539/copy-to-clipboard-in-node-js

Cross-platform clipboard module:
https://github.com/sindresorhus/clipboardy

**Comment by chriscalo** [comment]
Proposed usage in `index.js`:

``` js
import clipboardy from "clipboardy";
import { findPort } from "./find-port";

const isDev = process.env.NODE_ENV !== "production";

export async function start(server, startPort = 8000) {
  const PORT = process.env.PORT || await findPort(startPort);
  
  const listener = await server.listen(PORT, () => {
    console.log("");
    console.log(`App listening on port ${PORT}`);
    console.log("Press Ctrl+C to quit.");
    console.log("");
    
    if (isDev) {
      clipboardy.writeSync(`http://localhost:${PORT}/`);
    }
  });
  
  return listener;
};

export default start;
```

**Comment by chriscalo** [comment]
Alternative to avoid importing clipboardy in production:

``` js
import { findPort } from "./find-port";

const isDev = process.env.NODE_ENV !== "production";

export async function start(server, startPort = 8000) {
  const PORT = process.env.PORT || await findPort(startPort);
  
  const listener = await server.listen(PORT, () => {
    console.log("");
    if (isDev) {
      const clipboardy = await import("clipboardy");
      const url = `http://localhost:${PORT}/`;
      clipboardy.writeSync(url);
      console.log(`App listening at ${url}`);
      console.log(`URL copied to clipboard.`);
    } else {
      console.log(`App listening on port ${PORT}`);
    }
    console.log("Press Ctrl+C to quit.");
    console.log("");
  });
  
  return listener;
};

export default start;
```

**Comment by brillout** [comment]
Neat, I like this :+1:

**Comment by chriscalo** [comment]
Added copying URL to clipboard when `process.env.NODE_ENV !== "production"`.

Left to do:
- [x] Update `README.md` to explain this
- [x] Add test: when in dev, `http://localhost:8000/` is on clipboard
- [x] Add test: when in prod, clipboard is not modified
- [x] Publish to npm

**Comment by chriscalo** [comment]
Published v0.2.0:
https://www.npmjs.com/package/express-start
